tao_compiler/mlir/disc/transforms/lower_tf.cc

```diff
@@ -643,6 +651,30 @@ LogicalResult ConvertDequantizeOp::dequantize(TF::DequantizeOp op,
                          << "\n";
 }
 
+struct ConvertFakeQuantOp : public OpRewritePattern<TF::DiscFakeQuantOp> {
+  using OpRewritePattern::OpRewritePattern;
+
+  LogicalResult matchAndRewrite(TF::DiscFakeQuantOp op,
+                                PatternRewriter& rewriter) const final {
+    auto loc = op.getLoc();
+    auto input = op.input();
+    auto scale = op.scale();
+    auto zero_point = op.zero_point();
+    SmallVector<int64_t> axis;
+    for (auto& v : op.axis()) {
+      axis.push_back(v.cast<IntegerAttr>().getInt());
+    }
+    Value new_op = rewriter.create<mhlo_disc::FakeQuantOp>(
+        loc, input.getType(), input, scale, zero_point, op.use_signedAttr(),
+        op.use_symmetricAttr(), getI64ElementsAttr(axis, &rewriter),
+        op.num_bitsAttr(), op.quant_minAttr(), op.quant_maxAttr(),
+        op.use_dynamicAttr());
+    new_op.setType(op.output().getType());
```

Comment:
maybe we can remove this line after we changed line 668.

Reply:
done